multichain-api-gateway/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+  multichain-api-gateway:
+    ## you can build an image locally, or use pre-built images from registry
+    build: .
+    ports:
+      - "8044:8044"
+    environment:
+      ## optional: if provided, would be used as a configuration file
+      - MULTICHAIN_API_GATEWAY__CONFIG=/app/config.toml
```

Comment:
does this env works in your binary?

Reply:
Yes, tested by removing some instances in `config/base.toml` and making request